data/chebi_20/meta.yaml

```diff
@@ -0,0 +1,33 @@
+name: chebi_20
+description: A dataset of pairs of natural language descriptions and SMILEs.
+targets:
+- id: description
+  description: A natural language description of a SMILE
+  units: null
+  type: string
+  names:
+  - natural language description
+identifiers:
+- id: SMILES
+  type: SMILES
+  description: SMILES
+license: Y
+links:
+- url: https://aclanthology.org/2021.emnlp-main.47/
+  description: Original Text2Mol paper which introduced the dataset.
+- url: https://github.com/cnedwards/text2mol
+  description: Text2Mol data repository.
+- url: https://huggingface.co/datasets/OpenBioML/chebi_20
+  description: Hugging Face dataset for OpenBioML.
+num_points: null
```

Comment:
why null?

Reply:
Yes, I added the wrong key, it's changed to only having `num_points` now